lib/neo4j/active_node/persistence.rb

```diff
@@ -53,7 +72,7 @@ def create_model(*)
       node = _create_node(properties)
       init_on_load(node, node.props)
       send_props(@relationship_props) if @relationship_props
-      @relationship_props = nil
+      @relationship_props = @deferred_nodes = nil
```

Comment:
I guess so. It's really sensitive about the length of things, so I tend to try pushing out before down.